### ISSUE TITLE
Course.model.schema has old values for _accessibilityToggleTextOn and _accessibilityToggleTextOff

### ISSUE THREAD

**Issue by dancgray** [adaptlearning/adapt_framework#1450]
[_accessibilityToggleTextOn](https://github.com/adaptlearning/adapt_framework/blob/bc7ff5d315f9bd614dd3e5624fb4316fee568f08/src/core/schema/course.model.schema#L152) should be accessibilityToggleTextOn

[_accessibilityToggleTextOff](https://github.com/adaptlearning/adapt_framework/blob/bc7ff5d315f9bd614dd3e5624fb4316fee568f08/src/core/schema/course.model.schema#L163) should be accessibilityToggleTextOff

This will affect the translation of an export from the authoring tool for these values.

**Comment by moloko** [comment]
As it happens, [the code is set up to check for the underscored versions as well](https://github.com/adaptlearning/adapt_framework/blob/bc7ff5d315f9bd614dd3e5624fb4316fee568f08/src/core/js/views/accessibilityView.js#L26-L27) so that we didn't break backwards compatibility when we changed this - but we should of course get this corrected in the AT for consistency/correctness
